addons/website_forum/static/src/js/website_forum.js

```diff
@@ -257,31 +256,24 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         if (!karma) {
             return;
         }
-
         ev.preventDefault();
+        if (session.is_website_user) {
+            this._displayAccessDeniedNotification(Markup(_t('<p>Oh no! Please <a href="/web/login">sign in</a> to vote</p>')));
+            return;
+        }
         const forumID = parseInt(document.getElementById('wrapwrap').dataset.forum_id);
-        const notifOptions = {
+        const additionalInfoWithForumID = forumID
+            ? Markup`<br/>
+                <a class="alert-link" href="/forum/${encodeURIComponent(forumID)}/faq">
```

Comment:
`forumID` is an int, do we need to call `encodeURIComponent` ?

Reply:
yes, but we call `parseInt` just above so it can be trusted no ?

Reply:
You're absolutely, I read too fast. Thank you!